Update read me arks (#69)
bcgsc/arcs: README.md

# ARCS

Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. 
ARCS can be run in 2 modes:
* [ARCS](https://doi.org/10.1101/100750) (default) uses alignments of linked reads to the input contigs 

> * [ARKS](https://doi.org/10.1186/s12859-018-2243-x) (`--arks`) uses exact k-mer mapping associate linked reads to input contigs

Reviewer: ```suggestion
* [ARKS](https://doi.org/10.1186/s12859-018-2243-x) (`--arks`) uses exact k-mer mapping to associate linked reads to input contigs
```

> ### ARCS+LINKS Pipeline 

Author: I think explanation in this section holds for both methods that's why I didn't mention 'default' in the header or inside the section. Should I?

> ### ARCS+LINKS Pipeline 

Reviewer: Yes good point. Ok then leave this part as is, then you could add some information below on specifically how to run ARCS (with read alignments) vs. ARCS (arks mode)

An example bash script on how to run the ARCS+LINKS pipeline can be found at: Examples/pipeline_example.sh

### Running ARCS in default mode

Default mode used read alignments to contigs.

For running the pipeline default mode, Makefile should target `arcs` with `./arcs-make arcs`. For more info check `./arcs-make help`.

To run `./arcs` executable in default mode, run `./arcs` without additional flag. For argument requiriments check `./arcs --help`.

### Running ARCS in '--arks' mode

> ARKS is another scaffolding tool developed by our group designed based on kmerization paradigm. ARKS is integrated into ARCS(v1.x.x) for better usability.

Reviewer: Hey Murathan -- did this text get lost in the shuffle somewhere?

For running the pipeline with ARKS, Makefile should target `arks` with `./arcs-make arks`. For more info check `./arcs-make help`.

To run `./arcs` executable with ARKS, run `./arcs --arks`. For more info check `./arcs --help`.

### Demo

> you can test your installation by following instructions at: Examples/arcs_test-demo/README.txt

Reviewer: ```suggestion
You can test your installation by running one of our supplied demos:
* ARCS: `Examples/arcs_test-demo`
* ARKS: `Examples/arks_test-demo`
For both, you can compare your output to the files provided in the `output` folders within the above directories.
```

> for ARKS: Examples/arks_test-demo/README.txt

Reviewer: ```suggestion
```

> 

Reviewer: You can remove the rest of the demo text below here -- I think it is redundant with my suggestion above.

An example bash script on how to run the ARCS+LINKS pipeline can be found at: Examples/pipeline_example.sh

### Running ARCS in default mode

The default mode uses alignments of linked reads to contigs to scaffold the input contigs.

To run the pipeline in default mode, run `Examples/arcs-make arcs`. For more info check `Examples/arcs-make help`.

To run the `arcs` executable in default mode, run `arcs <alignments>`. For descriptions of all arguments, run `arcs --help`.

### Running ARCS in '--arks' mode

To run the pipeline in ARKS mode, run `Examples/arcs-make arks`. For more info check `Examples/arcs-make help`.

> To run the `./arcs` executable in ARKS mode, run `./arcs --arks`. For descriptions of all arguments, run `./arcs --help`.

Reviewer: ```suggestion
To run the `arcs` executable in ARKS mode, run `arcs --arks`. For descriptions of all arguments, run `arcs --help`.
```
Forgot to change this one too in my last round of suggestions to keep it consistent with ARCS

> You can test your installation by running one of our supplied demos:

Reviewer: There was a 'Demo' header here that got deleted accidentally I think.

Author: Oh I thought you wanted me to delete it. Putting it again
